app/src/ui/create-tag/create-tag-dialog.tsx

```diff
@@ -140,27 +130,35 @@ export class CreateTag extends React.Component<
     }
   }
 
-  private getCurrentError(sanitizedTagName: string): Error | null {
-    if (sanitizedTagName.length > MaxTagNameLength) {
-      return new Error(
-        `The tag name cannot be longer than ${MaxTagNameLength} characters`
+  private getCurrentError(): JSX.Element | null {
```

Comment:
I've moved this method to return `JSX` elements now, this way we can have nice rendering of tag names inside errors (it was a bit weird anyways that the method returned an `Error` object).